modules/installation-alibaba-config-yaml.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-alibaba-customizations.adoc
+
+[id="installation-alibaba-config-yaml_{context}"]
+= Sample customized `install-config.yaml` file for Alibaba Cloud
+
+You can customize the installation configuration file (`install-config.yaml`) to specify more details about
+your {product-title} cluster's platform or modify the values of the required
+parameters.
```

Comment:
@mtulio @jianli-wei @kwoodson @joelddiaz 
Can we get a sample install-config.yaml?